atomic_reactor/build.py

```diff
@@ -232,6 +233,7 @@ def recreate_parent_images(self):
 
     def set_base_image(self, base_image, parents_pulled=True, insecure=False):
         self.base_from_scratch = base_image_is_scratch(base_image)
+        self.custom_base_image = base_image_is_custom(base_image)
```

Comment:
this check should be done just once, before you set_base_image again with add_filesystem plugin to newly created image, or you should check if it was set already to True, you won't override it

it works for base_from_scratch because we are not calling set_base_image if base image is 'scratch'

Reply:
Fixed.